# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webauthn/WebAuthnTransactionChild.cpp
# Commit: 287bedb227e8
# Full Hash: 287bedb227e89933347633242efe1acd4236ad70
# Author: John Schanck <jschanck@mozilla.com>
# Date: 2025-02-12 09:32:07
# Regressor Bug: 1945969
# File Overlap Count: 1
# Description:
#   Bug 1945969 - part 5: rename WebAuthnManager to WebAuthnHandler. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D237257
# ==============================================================================

diff -r a01c165a2524 -r 287bedb227e8 dom/webauthn/WebAuthnTransactionChild.cpp
--- a/dom/webauthn/WebAuthnTransactionChild.cpp	Wed Feb 12 02:12:58 2025 +0000
+++ b/dom/webauthn/WebAuthnTransactionChild.cpp	Wed Feb 12 02:12:58 2025 +0000
@@ -4,21 +4,21 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "mozilla/dom/WebAuthnManager.h"
+#include "mozilla/dom/WebAuthnHandler.h"
 #include "mozilla/dom/WebAuthnTransactionChild.h"
 
 namespace mozilla::dom {
 
-void WebAuthnTransactionChild::SetManager(WebAuthnManager* aManager) {
-  mManager = aManager;
+void WebAuthnTransactionChild::SetHandler(WebAuthnHandler* aHandler) {
+  mHandler = aHandler;
 }
 
 void WebAuthnTransactionChild::ActorDestroy(ActorDestroyReason why) {
   // Called by either a __delete__ message from the parent, or when the
   // channel disconnects. Clear out the child actor reference to be sure.
-  if (mManager) {
-    mManager->ActorDestroyed();
-    mManager = nullptr;
+  if (mHandler) {
+    mHandler->ActorDestroyed();
+    mHandler = nullptr;
   }
 }
 